docs/DICTIONARIES.md

```diff
@@ -31,6 +31,7 @@ This project uses `dictionaries` to unify props between different components.
 | ----------------- | ---------------------------------------------------------------- | --------------- |
 | Action Color      | `primary`, `secondary`, `tertiary`, `inverted`                   | ActionColor     |
 | Action Link Color | `primary`, `secondary`, `inverted`                               | ActionLinkColor |
+| Background Color  | `primary`, `secondary`, `tertiary`                               | BackgroundColor |
 | Emotion Color     | `success`, `informative`, `warning`, `danger`                    | EmotionColor    |
 | Text Color        | `primary`, `secondary`, `primary-inverted`, `secondary-inverted` | TextColor       |
```

Comment:
**Off topic:** This still need to be reviewed, v3 is out and this is still outdated… 🙈 

@crishpeen is it https://jira.almacareer.tech/browse/DS-1545 ? We should queue up this one ASAP…

cc @literat